build.sbt

```diff
@@ -179,13 +194,37 @@ lazy val docs = project("docs")
     }
   )
 
+lazy val bindings = project("bindings")
+  .settings(
+    publish / skip := false
+  )
+  .aggregate(
+    libposix,
+    libutf8proc
+  )
+
+lazy val libposix = bindingProject("posix")
+  .configure(binding("fnmatch.h"))
+  .configure(binding("iconv.h"))
```

Comment:
This shows how you can generate multiple objects per sbt project. `iconv.h` is not actually POSIX but instead UNIX98. Not sure if we can come up with a better name, like `libc`?!

Reply:
Can we place in separate `iconv` project?